packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -70,10 +72,9 @@ exports.onPostBuild = (args, pluginOptions) => {
     globDirectory: rootDir,
     globPatterns,
     modifyUrlPrefix: {
-      rootDir: ``,
```

Comment:
I'm usually of the opinion to just keep things as/is if I don't _know_ what something is doing. I'll check this out locally and validate.

Reply:
https://developers.google.com/web/tools/workbox/modules/workbox-build#full_generatesw_config 

this would translate: if there's a path that starts with the *string* rootDir, replace with nothing. this is potentially even a bug if i have a file named "rootDir" but it will probably not be captured at all due to the missing slash